test/jest/Bladeburner/BladeburnerCasualties.test.ts

```diff
@@ -0,0 +1,90 @@
+import { OperationTeam } from "../../../src/Bladeburner/Bladeburner";
+
+jest.mock("../../../src/utils/helpers/getRandomIntInclusive");
+
+import {
+  CasualtyReport,
+  KillBladeburnerCasualties,
+  OperationCasualtyOutcome,
+} from "../../../src/Bladeburner/Actions/Casualties";
+
+import { getRandomIntInclusive } from "../../../src/utils/helpers/getRandomIntInclusive";
+
+describe("Bladeburner Team", () => {
+  beforeEach(() => {
+    // Return the high end of all random rolls
+    getRandomIntInclusive.mockImplementationOnce((_: number, high: number) => high);
```

Comment:
Why not just use `mockImplementation`? You're not doing something complicated such that the "Once" part is important.

Reply:
Since it's not injected, I want the test to fail in case the underlying code ever gets expanded with more than one use.